src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -661,7 +678,7 @@
         float: right;
         vertical-align: middle;
         @include equilateral($clr-datagrid-icon-size);
-        margin: 0 $clr_baselineRem_0_25;
+        margin-left: $clr_baselineRem_0_25;
```

Comment:
This was a judgement call to put the filter toggle buttons visually as close to where they used to be.